### ISSUE TITLE
Instructions seems a bit outdated

### ISSUE THREAD

**Issue by kant2002** [hermitcore/rust-std-hermit#8]
I try to compile Hermit on Windows, and expereince issue building custom std.
In the instruction there mentioned `install.sh`. I see only `dist.sh` and `install-template.sh`. Also there some unknown `HERMIT_TOOLCHAIN` variable, so I cannot recreate steps myself. Can you advice on the direction to move forward.

**Comment by mkroening** [comment]
You need to follow step one and download a release artifact. Those will contain `install.sh` generated from `install-template.sh`.

Does that work for you? Should we clarify in the README? If so, how? :)

**Comment by kant2002** [comment]
Thank, it give me some ideas. Since I'm on Windows Windows, not on WSL, I have to build std myself seems to be. Please correct me if I still can/should use existing files from this repo. If not, below I will try to explain where I get stuck.

Currently I'm fighting `./x.ps1 dist ...` and environment setup. 
https://github.com/kant2002/rust-std-hermit/actions/runs/11553105844 for example.

**Comment by mkroening** [comment]
Native Windows works, but we only supply bash installation scripts right now. You could use git's bash, for example.

**Comment by mkroening** [comment]
See https://github.com/hermit-os/hermit-rs-template/actions/runs/10725649523/job/29743981018 for example.

**Comment by KairusZhang** [comment]
> You need to follow step one and download a release artifact. Those will contain `install.sh` generated from `install-template.sh`.
> 
> Does that work for you? Should we clarify in the README? If so, how? :)

Yes , there are some misunderstanding for downloading rust-std-1.83.0-x86_64-unknown-hermit.tar.gz  Rather than  source code.zip.
